lldb/source/Plugins/Process/Dpu/Dpu.cpp

```diff
@@ -444,7 +444,7 @@ StateType Dpu::StepThread(uint32_t thread_index, unsigned int *exit_status) {
   return StateType::eStateStopped;
 }
 
-bool Dpu::WriteWRAM(uint32_t offset, const void *buf, size_t size) {
+bool Dpu::WriteWRAMTreatingAlignment(uint32_t offset, const void *buf, size_t size) {
```

Comment:
I'm not sure that "Treating" is how English would put it. See https://www.linguee.com/english-french/translation/treat.html